Tests/SwiftFixItTests/BasicTests.swift

```diff
@@ -284,25 +292,25 @@ final class BasicTests: XCTestCase {
                     PrimaryDiagnostic(
                         level: .error,
                         text: "error1",
-                        location: .init(filename: filename1, line: 1, column: 1, offset: 0),
+                        location: .init(filename: filename1, line: 1, column: 5, offset: 0),
```

Comment:
question:  could you please explain how a change in the test from XCTest to Swift Testing could cause a difference in the test data?

Reply:
There was no difference. I spotted that the test was fragile because the diagnostics had identical fix-its and didn’t bother to extract the tweak because they would all most likely be squashed upon merge. Happy to leave it out if you want me to.